### ISSUE TITLE
input.textarea中设置style高度为固定像素时,resize事件对外边框无效

### ISSUE THREAD

**Issue by Jiong-Sen** [ant-design/ant-design#41947]
### Reproduction link

[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/dai-zi-shu-ti-shi-de-wen-ben-yu-antd-5-4-1-forked-xfleyr?file=/demo.tsx)

### Steps to reproduce

拖动右下角resize的icon,上下拖动

### What is expected?

外边框和内部元素高度一致

### What is actually happening?

外边框和内部元素高度不一致

| Environment | Info |
| --- | --- |
| antd | 5.4.4 |
| React | react18 |
| System | macos |
| Browser | 112.0.5615.137 |

---
在antd 5.3.2版本中是正常的

**Comment by MadCcc** [comment]
5.4.0 修改了边框样式所在的元素，原本是在 textarea 上的，看起来需要兼容一下。目前版本可以这样：
https://codesandbox.io/s/dai-zi-shu-ti-shi-de-wen-ben-yu-antd-5-4-1-forked-odyx0h?file=/demo.tsx

**Comment by Francoshum95** [comment]
Let me have a look ~~

**Comment by Francoshum95** [comment]
I need to pass this on to others who are familiar with the repo. 
So I did my best to look into both the [AntDesign](https://github.com/ant-design/ant-design) and [rc-textarea](https://github.com/react-component/textarea) repos, and I found that the inline style in the `<span>` element is not updating like the `<textarea>` element does. I tried to find the logic that controls the resizing value and put it into the <span> element, but I could not find it (maybe it's handled by [rc-resize-observer](https://github.com/react-component/resize-observer)). So, I am passing this along to someone who may be familiar with the above repo. Cheers!
